letsencrypt/error_handler.py

```diff
@@ -1,5 +1,6 @@
 """Registers functions to be called if an exception or signal occurs."""
 import logging
+import sys
```

Comment:
nit: we order imports alphabetically (and group them by stdlib/3rd-party/local sections)